recipes/fsnviz/meta.yaml

```diff
@@ -1,29 +1,29 @@
 package:
   name: fsnviz
-  version: 0.1.0
+  version: 0.2.0
 
 source:
-  fn: FsnViz-0.1.0.tar.gz
-  url: https://pypi.python.org/packages/cb/ed/a9cd233386fab2800e24a5a5b58757c150e35aa0bb30069e44498db4b6a3/FsnViz-0.1.0.tar.gz
-  md5: 79ba80aeef411b89802e2b3e597b3c37
+  fn: 33816ab113d642f0c123e99ac5ac47359eea414a.tar.gz
+  url: https://github.com/bow/fsnviz/archive/33816ab113d642f0c123e99ac5ac47359eea414a.tar.gz
+  md5: 3096b741fe9b575a9d0292dcad508e80
 
 build:
-  skip: True # [py2k or py34]
+#  skip: True # [py2k or py3]
   number: 0
-  entry_points:
-    - fsnviz=fsnviz.main:cli
+#  entry_points:
+#    - fsnviz=fsnviz.main:cli
 
 requirements:
   build:
-    - python
+    - python >=3.4
     - setuptools
     - click >=6.6
     - crimson >=0.3.0
     - jinja2 ==2.8
```

Comment:
Jinja2 should be in conda-forge by now. Would you mind updating this requirement string to `jinja2 ==2.9.5`?

Also, adding `circos >=0.69.2` would be useful.